app/src/docs/v1.api-spec.yaml

```diff
@@ -17,30 +17,348 @@ security:
   - BearerAuth: []
     OpenID: []
 paths:
-  /hello:
+  '/':
```

Comment:
All of these paths do not start at the root level - they are at `/object/...` level.